ls1intum/Artemis
# `Iris`: Call API to accept Iris policy

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

### Motivation and Context
Call API to accept Iris policy and get the accepted timestamp from the server

### Description


### Steps for Testing

Prerequisites:
- 1 Student that hasn't accepted Iris policy yet
- 1 Programming Exercise with Complaints enabled

1. Log in to Artemis
2. Go to a programming exercise with Iris enabled
3. Click on Iris icon and should be prompted to accept the policy
4. Accept the policy and a welcome message will be shown
5. Reload the page and the welcome message should still be there
6. Chat something
7. Reload the page and the welcome message should be disappeared

#### Exam Mode Testing



### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage







### Screenshots
